File: MPI/mpicheck/tests/functions/4.0/MPI_Session_get_pset_info.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Session var_0;
    const char *var_1;
    MPI_Info *var_2;
    int ret;
    /* calls */
    ret = MPI_Session_get_pset_info(var_0, var_1, var_2);
    ret = PMPI_Session_get_pset_info(var_0, var_1, var_2);
    return 0;
}

File: MPI/mpicheck/tests/functions/4.0/MPI_Session_get_pset_info.f

        
        program main
        include 'mpif.h'
        
        INTEGER var_0
       CHARACTER*(10) var_1
       INTEGER var_2
       INTEGER var_3
        call mpi_session_get_pset_info(var_0, var_1, var_2, var_3)
       call pmpi_session_get_pset_info(var_0, var_1, var_2, var_3)
        end program main
    